swagger.yaml

```
@@ -510,10 +547,17 @@ definitions:
         type: integer
         description: "Image height, in number of pixels"
         example: 1080
-      private:
+      state:
         type: string
-        description: "S3 Private bucket name"
-        example: "my-private-bucket"
+        description: "The state of this download variant. The high level image will change its state to imported only when all variants are in imported state."
+        enum:
+          - pending
+          - importing
+          - imported
+          - published
+          - completed
+          - failed
```

As discussed, if we are creating more specific and consistent states then this needs to be:

```suggestion
          - failed-import
          - failed-publish
```